.github/workflows/ci.yml

```diff
@@ -1227,10 +1123,8 @@ ${{ hashFiles('.pre-commit-config.yaml') }}"
       RUNS_ON: ${{ fromJson(needs.build-info.outputs.runsOn) }}
       PYTHON_MAJOR_MINOR_VERSION: ${{ matrix.python-version }}
       CURRENT_PYTHON_MAJOR_MINOR_VERSIONS_AS_STRING: ${{needs.build-info.outputs.pythonVersionsListAsString}}
-    # Only run it for direct pushes
-    if: >
-      github.ref == 'refs/heads/main' || github.ref == 'refs/heads/v1-10-test' ||
-      github.ref == 'refs/heads/v2-0-test' || github.ref == 'refs/heads/v2-1-test'
+    # Only run it for direct pushes and scheduled builds
+    if: github.event_name == 'push' || github.event_name == 'schedule'
```

Comment:
This will run if any committer pushes a branch to apache/airflow instead of their own fork -- and I don't think we want that to happen.

Reply:
Not really :). I was looking on how to generalize that, but I noticed that there is the "push" if at the top of the file:

```
  push:
    branches: ['main', 'v[0-9]+-[0-9]+-test']
```

Reply:
I can split it into several ones, no problem

Reply:
Ah cool!